packages/modular-scripts/src/__tests__/start-app.tsx

```diff
@@ -0,0 +1,120 @@
+import execa from 'execa';
```

Comment:
This should be within a utils folder in `__tests__` so that it's clear it's not a test. Also the extension should be `.ts`

Reply:
It's clear it's not a test because it doesn't have the `.test` extension - otherwise we'll end up putting everything in `utils` at this rate!

Reply:
And also I just realized there's already a utiils folder within __tests__ so that would not work. Just the extension change then!